armi/reactor/tests/test_flags.py

```diff
@@ -27,8 +27,16 @@ def test_fromString(self):
         self.assertEqual(flags.Flags.fromStringIgnoreErrors("invalid"), flags.Flags(0))
 
     def test_toString(self):
+        """
+        Convert flag to and from string for serialization.
```

Comment:
I like what you're doing here.

Maybe we should change the NAME of this test so it says "to" and "from"?

`test_flagsToAndFromString()` or something like that?